src/web/app/components/copy-course-modal/copy-course-modal.component.scss

```diff
@@ -0,0 +1,7 @@
+hr.solid-divider {
+  border-top: 1px solid #BBB;
```

Comment:
Can override the margin here? The default margin is way too much. Something like 0.3rem?

Reply:
I have reduced the margin for the border to look like this:
![ezgif com-gif-maker](https://user-images.githubusercontent.com/58677983/122697317-a8363c80-d277-11eb-8f4d-a4dd60163a19.gif)